sass/blocks/developer/_developer.scss

```diff
@@ -11,27 +11,20 @@
     text-transform: uppercase;
   }
 
-  &__logo {
+  &__svg {
```

Comment:
ok, I don't think it's a good idea to to apply so many properties to svg. You have a parent link, use this for styiling.